packages/jaeger-ui/src/components/DeepDependencies/Graph/index.css

```diff
@@ -14,16 +14,22 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-.DdgArrow.is-pathHovered {
+.Ddg--Edges {
+  stroke: #444;
+  stroke-width: 0.7;
+}
+
+.Ddg--Edges.is-withViewModifiers {
```

Comment:
this seems to accomplish the `is-notMatched` but now seems a little less clear IMO

Reply:
I see, it's to apply one class to all `Edges` not a class to every `Edge`. Missed the `s` at first

Reply:
I would call this `.Ddg--Edges.contains-emphasis` because view modifiers other than emphasis shouldn't cause all edges to have their stroke opacity lowered.

Reply:
not worth the performance hit to distinguish between VMs that necessitate de-emphasizing other edges and (non-existent) VMs that don't. worth revisiting when other edge VMs are introduced.